### PR TITLE
Transport

### PR DESCRIPTION
**Pull request by borisyazhov**

### CODE REVIEW

File: src/com/company/Transport/dpo/transport/TransportCreator.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+    @Override
+    public void setNumber(String number) { this.number = number; }
+
+    @Override
+    public PaymentSystem getPay() { return pay; }
+
+    @Override
+    public void setPay(PaymentSystem pay) { this.pay = pay; }
+
+    public void GoOutOf(Station station, int index) {
```
Comment (secourse2019) [inline src/com/company/Transport/dpo/transport/TransportCreator.java]: use camelCase for methods

File: src/com/company/Transport/util/Pair.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.company.Transport.util;
+
+public class Pair<First, Second> {
+
+    public First first;
+    public Second second;
+
+    Pair(First first, Second second) {
```
Comment (secourse2019) [inline src/com/company/Transport/util/Pair.java]: use Map instead of pair

**Review by secourse2019 — CHANGES_REQUESTED** [review]
not bad, some comments

**Review by Kjatata — APPROVED** [review]

File: src/com/company/Transport/dpo/departmant/Garage.java [hunk 3]
```diff
@@ -1,25 +1,26 @@
...
         System.out.println("Fixed all problems");
     }
 
-    public ArrayList<TransportCreator> getTransportList() {
+    public AbstractList<TransportCreator> getTransportList() {
         return transportList;
     }
 
-    public void setTransportList(ArrayList<TransportCreator> transportList) {
+    public void setTransportList(AbstractList<TransportCreator> transportList) {
```
Comment (secourse2019) [inline src/com/company/Transport/dpo/departmant/Garage.java:23]: just List will be enough

**Review by secourse2019 — APPROVED** [review]
good job

**Pull request merged**